### PR TITLE
PROJQUAY-3291:Added pg_trgm requirements

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Issue: https://issues.redhat.com/browse/PROJQUAY-3291

Preview: https://snarayan-redhat.github.io/quay-docs/PROJQUAY-3291_pgtrgm/index.html#operator-unmanaged-postgres

### CODE REVIEW

File: modules/operator-managed-postgres.adoc [hunk 1]
```diff
@@ -1,6 +1,17 @@
 [[operator-managed-postgres]]
 = Using the managed PostgreSQL
 
+Requirements:
+
+If you are using an externally managed PostgreSQL database, you must manually enable `pg_trgm` extension for a successful deployment.
```
Comment (stevsmit) [inline modules/operator-managed-postgres.adoc:6]: Just a few small errors I've addressed below: 


If you are using an externally managed PostgreSQL database, you must manually enable `pg_trgm` extension for a successful deployment.
Enter the following command to enable the `pg_trgm` extension, replacing the name of the PostgreSQL pod with your pod: 
[source,terminal]
$ oc exec -it postgres-xxxxxxxxxx-xxxxx -n quay-enterprise -- /bin/bash -c 'echo "CREATE EXTENSION IF NOT EXISTS pg_trgm" | /opt/rh/rh-postgresql10/root/usr/bin/psql -d quay'

[NOTE]
====
The `-d`, or database name, must not be omitted. If omitted, the extension will be created on the default PostgreSQL database.
====

**Comment by dongboyan77** [comment]
I think the updates should be added in sub-title `3.3.1. Using an existing Postgres database`. Since the bug description is for external database instead of managed database by operator. The database managed by operator has been added pg_trgm extension by default, it is not needed to add manually.
So that means oc command is also not suitable, because customer's external database maybe cloud database service like AWS RDS, it is not installed in openshift. I suggest to remove the command.

**Comment by stevsmit** [comment]
@dongboyan77 " . . . I suggest to remove the command."

Are you suggesting:

1. We move the command under a new section `3.3.1 Using an existing Postgres database` for external databases?

**OR**

2. Not including it at all? 


Sorry, we are a little confused. 

Thanks

**Comment by dongboyan77** [comment]
I mean 

> Not including it at all

Only add requirements in 3.3.1 section

**Comment by dongboyan77** [comment]
The key for this issue is reminding customer to create 'pg_trgm' extension instead of what command is. Why remove the requirement and only remain the command.
Your original commit is available, I re-paste here:

> Requirements:

If you are using an externally managed PostgreSQL database, you must manually enable `pg_trgm` extension for a successful deployment.

**Comment by snarayan-redhat** [comment]
> The key for this issue is reminding customer to create 'pg_trgm' extension instead of what command is. Should not remove the requirement. Your original description is available, I re-paste here:

@dongboyan77 Is this fine now?

**Comment by dongboyan77** [comment]
Hey, we are talking about 

> 3.3.1. Using an existing Postgres database
You modified wrong section

**Comment by snarayan-redhat** [comment]
> Hey, we are talking about
> 
> > 3.3.1. Using an existing Postgres database
> 
> You modified wrong section

My bad! Does it look good now?

**Comment by dongboyan77** [comment]
LGTM

**Pull request merged**